ermia_takada/frame.cc

```diff
@@ -164,10 +164,11 @@ void displayParameter()
 
 void makeDB()
 {
-    posix_memalign((void **)&Table, PAGE_SIZE, (tuple_num) * sizeof(Tuple));
+    [[maybe_unused]] auto err = posix_memalign((void **)&Table, PAGE_SIZE, (tuple_num) * sizeof(Tuple));
     for (int i = 0; i < tuple_num; i++)
     {
-        Table[i].key = 0;
+        new (&Table[i]) Tuple();
+        Table[i].key = i;
```

Comment:
L170 invokes constructor of `Tuple` via placement new (to set zero to `tuple.mmt.counter` locking)
L171 is fixing